portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,62 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Gochi+Hand&family=Manrope&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <header id="intro">
+            <h3>Hi I'm</h3>
+            <h2 id="name"> Rebecca Sun</h2>
+        </header>
+
+        <h4 id="categories">-ABOUT-</h4>
+        <div id="about">
+            <img id="headshot" src="images/me.jpg"></img>
+            <p>Currently, I am a <a href="https://buildyourfuture.withgoogle.com/programs/step/">STEP</a> intern at Google. 
+            I am a rising junior at Vanderbilt University majoring in computer science with a minor in engineering management. 
+            This summer, I'm working out of Kansas City, Kansas where I'm from.</p>
+        </div>
+        
+        <h4 id="categories">-EXPERIENCE-</h4>
+        <ul id="info">
```

Comment:
Same thing for these <ul> elements as what I mentioned above... use class="info" or class="info-list" maybe to be more specific instead of having an id.  (here, and the other lists below where you are also using id="info")